libs/langchain/langchain/chat_loaders/imessage.py

```diff
@@ -46,6 +46,36 @@ def __init__(self, path: Optional[Union[str, Path]] = None):
                 "Please install it with `pip install pysqlite3`"
             ) from e
 
+    def _parse_attributedBody(self, attributedBody: bytes) -> str:
```

Comment:
could we add a simple unit test for this function

Reply:
Thank you for the suggestion, a simple unit test function has been added now.